src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateExecution.java

```diff
@@ -81,9 +82,14 @@ protected ExternalWorkspace run() throws Exception {
         } else {
             // this is the downstream job
 
-            Item upstreamJob = findUpstreamJob(upstreamName);
+            if (step.getDiskPoolId() != null) {
+                listener.getLogger().println("WARNING: Both 'upstream' and 'diskPoolId' parameters were provided. " +
+                        "The 'diskPoolId' parameter will be ignored. The step will allocate the workspace used by the upstream job.");
```

Comment:
@oleg-nenashev @martinda WDYT about this warning message? Is it understandable for the user?